Sources/EmbraceCore/Embrace.swift

```diff
@@ -3,6 +3,7 @@
 //
 
 import Foundation
+import OpenTelemetrySdk
```

Comment:
Did we care if this is here or should we move this and its usage to another file?

Reply:
I do, so I moved it.